website/docs/r/main_route_table_association.html.markdown

```diff
@@ -9,6 +9,8 @@ description: |-
 # Resource: aws_main_route_table_association
 
 Provides a resource for managing the main routing table of a VPC.
+It is recommended you **do not** use both `aws_default_route_table` to manage the default route table and use the `aws_main_route_table_association`, due to possible conflict in routes.
+For more context see [aws_default_route_table][tf-default-route-table].
```

Comment:
Rather than these extra lines, I think it would be better to keep a note section using the same language as the `aws_default_route_table`:

```suggestion

~> **NOTE:** **Do not** use both `aws_default_route_table` to manage a default route table **and** `aws_main_route_table_association` with the same VPC due to possible route conflicts. See [aws_default_route_table][tf-default-route-table] documentation for more details.
```